docs/UserGuide.md

```diff
@@ -39,96 +39,166 @@
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+
+Example:
+* `help` 
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a floating task: `add`
+Adds a task to the todo list<br>
+Format:`add TASK_NAME [tag/TAG]...` 
+
+> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
+> items with `...` after them can have multiple instances. Order of parameters are fixed. 
+>  
+> Tasks can have any number of tags (including 0)
+
+Examples: <br>
+* `add Homework`<br>
+* `add Homework tag/CS1231`
+
+#### Adding a task with deadline: �add�
+Format: `add TASK_NAME end/DATE TIME [tag/TAG]...`
```

Comment:
Consider supporting shorter forms of keywords. Advanced users should be able to do the same actions with fewer key strokes.